proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -528,7 +528,9 @@ public static VelocityConfiguration read(Path path) throws IOException {
                     "The file " + forwardSecretFile + " is not a valid file or it is a directory.");
           }
         } else {
-          throw new RuntimeException("The forwarding-secret-file does not exist.");
```

Comment:
I would probably print something at the very least, just for the sake of debugging certain types of setups